module/localisation/FieldLocalisation/src/FieldLocalisation.cpp

```diff
@@ -221,6 +200,32 @@ namespace module::localisation {
                 }
             }
 
+            // Left side penalty mark
+            cfg.initial_state.emplace_back((fd.dimensions.field_length / 2.0),
+                                           (fd.dimensions.field_width / 2.0),
+                                           -M_PI_2);
+
+            // Right side penalty mark
+            cfg.initial_state.emplace_back((fd.dimensions.field_length / 2.0),
+                                           (-fd.dimensions.field_width / 2.0),
+                                           M_PI_2);
```

Comment:
if we're specifying that positive half is our starting position, we should document this somewhere. in the readme, nubook, or a comment here